templates/course/information.php

```diff
@@ -0,0 +1,20 @@
+<?php
+global $post;
+$course = new ADM\WPPlugin\PostTypes\Course();
+$metas = $course::$metas;
+if (!empty($metas)) : ?>
+    <ul class='admwpp-meta-information'>
+        <?php foreach ($metas as $key => $value) :
+            $label = $value['label'];
+            $type = $value['type'];
+            $showOnFornt = $value['showOnFront'];
+            $tmsKey = $value['tmsKey'];
+            $value = get_post_meta($post->ID, $key, true);
```

Comment:
move this to under the condition, no need to get the post meta if we don't need to display it :)